ca/templates/_layout.html

```diff
@@ -30,6 +30,6 @@ <h2>
 
       </div>
     </div>
-
+    ca.berlin.freifunk.net wird von der Berliner Freifunk-Gemeinschaft <a href="https://github.com/freifunk-berlin/ca.berlin.freifunk.net">auf GitHub</a> entwickelt. Du kannst mithelfen und die Software verbessern.
```

Comment:
to hard-code "ca.berlin.freifunk.net" ist eher unpraktisch. Denn der Code wird auch bei "tunnel.berlin.freifunk.net" genutzt.

P.S: ich dachte, ich schrieb das schon mal ....

Reply:
Vorschlag: " ca.berlin.freifunk.net" ändern in "dieser Web-service"